src/pages/index.js

```diff
@@ -65,34 +166,65 @@ consts.profileEditButton.addEventListener("click", () => {
 consts.profileAddButton.addEventListener("click", () => {
   addCardModal.open();
 });
+consts.avatarContainer.addEventListener("click", () => {
+  avatarEditModal.open();
+});
 
-// validation
-
-const addCardValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.addCardModalDiv,
-);
-
-addCardValidator.enableValidation();
-
-const profileEditValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.profileEditDiv,
-);
+// initial loading handlers
 
-profileEditValidator.enableValidation();
+function handleProfileLoad({ name, about, avatar }) {
+  profileEl.setUserInfo(name, about);
+  profileEl.setAvatar(avatar);
+}
+function loadAll() {
+  api
+    .initialLoad()
+    .then((data) => {
+      handleProfileLoad(data[0]);
+      return data;
+    })
+    .then((data) => {
+      loadDefaultCards(data[1]);
+    });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**